sys/kern/src/arch/arm_m.rs

```diff
@@ -309,15 +309,14 @@ pub fn reinitialize(task: &mut task::Task) {
         .iter()
         .find(|region| region.contains(initial_stack.saturating_sub(4)))
     {
-        let mut uslice: USlice<u32> = USlice::from_raw(
+        if let Ok(mut uslice) = USlice::<u32>::from_raw(
```

Comment:
mmmaybe worth a comment here to the effect of "if this fails, don't crash the whole system, as it's just for diagnostic purposes"? also, perhaps we should mention why it might fail here?